ntbs-service/Pages/Notifications/NotificationModelBase.cs

```diff
@@ -20,15 +21,16 @@ public abstract class NotificationModelBase : PageModel
             INotificationRepository notificationRepository = null)
         {
             this.Service = service;
-            this.NotificationRepository = notificationRepository;
             this.AuthorizationService = authorizationService;
+            this.NotificationRepository = notificationRepository;
         }
 
         protected NotificationGroup Group;
         public int NumberOfLinkedNotifications { get; set; }
 
         public Notification Notification { get; set; }
         public NotificationBannerModel NotificationBannerModel { get; set; }
+        public IList<Alert> Alerts { get; set; }

```

Comment:
Does the currently layout structure mean that Alerts will be non-trivial to add on any of the edit pages? If so then I think IList<Alerts> could probably be restricted to the overview page?